qtpy/tests/test_qtserialbus.py

```diff
@@ -0,0 +1,28 @@
+"""Test QtSerialBus."""
+
+import pytest
+from qtpy import (
+    parse,
+    PYSIDE6,
+    PYSIDE_VERSION,
+    QtBindingMissingModuleError,
+)
+
+
+@pytest.mark.skipif(not PYSIDE6 or parse(PYSIDE_VERSION) >= parse('6.5'), reason='.')
```

Comment:
I presume you're going to fill the `reason` in later?

Reply:
Yup!